presto-parser/src/main/java/com/facebook/presto/sql/tree/SimpleGroupBy.java

```diff
@@ -42,7 +42,7 @@ public SimpleGroupBy(NodeLocation location, List<Expression> simpleGroupByExpres
     private SimpleGroupBy(Optional<NodeLocation> location, List<Expression> simpleGroupByExpressions)
     {
         super(location);
-        this.columns = requireNonNull(simpleGroupByExpressions);
+        this.columns = ImmutableList.copyOf(requireNonNull(simpleGroupByExpressions, "simpleGroupByExpressions is null"));
```

Comment:
IMO, this style makes the code less readable vs doing the requireNoNullCheck as a separate step.